src/appshell/view/appmenumodel.cpp

```diff
@@ -114,6 +137,17 @@ void AppMenuModel::setupConnections()
         MenuItem& pluginsItem = findMenu("menu-plugins");
         pluginsItem.setSubitems(makePluginsMenuSubitems());
     });
+
+    context()->currentNotationChanged().onNotify(this, [this]() {
+        auto stack = undoStack();
+        if (stack) {
+            stack->stackChanged().onNotify(this, [this]() {
+                updateUndoRedoItems();
+            });
+        } else {
+            updateUndoRedoItems();
+        }
```

Comment:
Small thing:
```suggestion
        auto stack = undoStack();
        if (stack) {
            stack->stackChanged().onNotify(this, [this]() {
                updateUndoRedoItems();
            });
        }
        
        updateUndoRedoItems();
```
(initially in my UndoRedoModel PR I had the same, but then made this same change in a subsequent push)

Reply:
I made the suggested change as well as re-merging your pull request to pick up your similar change. I had made a change to that very routine in UndoRedoModel to eliminate code duplication, but I think I picked up the relevant change from your PR.